### PR TITLE
Create chaingpt-research.md

### LINKED ISSUE #139: Growth Hacking Case Study: ChainGPT
Stumbled upon https://www.chaingpt.org/ and I think their growth strategy is insane. I am highly, highly skeptical about their technology but looking at the way the company and the founder presents himself is top notch for a new startup. 

What really put me over the edge for being impressed was the "as seen on" section on their homepage. They have linked proof of real tweets and press releases from respected names about their project. 

I think we should study them. 

### Deliverable

Outline everything they did from launch to now. Make a plan and then we can replicate it in the form of GitHub tasks under a new repo.

### PR DESCRIPTION
**Pull request by sergfeldman**
Resolves #139

### CODE REVIEW

**Review by 0x4007 — CHANGES_REQUESTED** [review]
This overview is nice but is not ready because it lacks the second half of the prompt: 

> Make a plan and then we can replicate it in the form of GitHub tasks under a new repo.

**Comment by sergfeldman** [comment]
@pavlovcik I will be very happy if we achieve such success as ChainGPT. I`ve added a preliminary plan from my point of view.

File: chaingpt-research.md [hunk 1]
```diff
@@ -18,6 +18,29 @@ Each company has its own unique path to success. Repeating the actions of anothe
...
+Preliminary steps to leverage ChainGPT's expertise
+
+- Alex reviews the collected information about the ChainGPT founder.
+After this, Alex and Sergey determine what experience can be applied to Ubiquity.
+Consider the option of revealing the founder's identity and greater publicity.
+
+- Alex reviews the collected information about ChainGPT's marketing and website.
+After this, Alex and Sergey determine what experience can be applied to Ubiquity.
+Consider options for cooperation with marketing agencies similar to those that supported ChainGPT.
+To think that for such success as ChainGPT, we need to focus on a hype-able product, and not necessarily on serious development.
```
Comment (0x4007) [inline chaingpt-research.md:30]: Well I think that we spent a fair amount of time developing the fundamentals of our business. Now we can embellish the truth and "sell the dream" which is common in sales and marketing.

Think of something like WeWork. They had a real business (albeit not profitable as far as I recall) but Adam Neumann made the idea of coworking spaces sound like something a lot more exciting than office spaces.

They were valued at over a billion dollars for some office space rentals lol

**Review by 0x4007 — COMMENTED** [review]
It seems like a lot of this is bottlenecked by me reviewing. Instead this task was intended for the researcher to provide proposals. I would review the proposals, not the raw research.

**Comment by sergfeldman** [comment]
> It seems like a lot of this is bottlenecked by me reviewing. Instead this task was intended for the researcher to provide proposals. I would review the proposals, not the raw research.

@pavlovcik I understand your request to receive a ready-made list of proposals as a result of research.
I think that this will not bring a positive result, because it is not incremental specific actions that are important, but the company strategy as a whole.
To understand the ChainGPT strategy, we need to dive a little deeper into it.

However, I have added key actions that I think are relevant to us.
Please let me know if you want anything else.

**Comment by 0x4007** [comment]
> I think that this will not bring a positive result, because it is not incremental specific actions that are important, but the company strategy as a whole.

I don't understand what you mean. Also, wouldn't it have made more sense to say this on the issue before starting the project if you think it doesn't make sense?

> but the company strategy as a whole.

I think I understand now. The purpose of this task is to use them as a case study and then create an actionable set of actions for us that we can apply that makes sense in the context of Ubiquity.

**Comment by sergfeldman** [comment]
@pavlovcik My thoughts on comparing the approaches of Ubiquity and ChainGPT.
Ubiquity has a more rational and substantial approach, and ChainGPT is perceived as more innovative and emotional.
Certain actions make sense only in the context of a specific approach.

A few examples:

ChainGPT was accelerated by Seedify and this step is crucial for their growth.
Ubiquity did not go through acceleration.
It would be possible to add the task “Complete acceleration in Seedify” to the actionable plan.

The founder of ChainGPT is public and actively establishes partnerships. 
It would be possible to add the task “The founder should go public and actively establish partnerships.”

But I don't think it makes sense for Ubiquity to repeat these specific actions, because we have a different approach.
The top 5 recommendations that are relevant to us were added in the commit https://github.com/ubiquity/research/pull/140/commits/d7dfe1ed5a2e28c9e51755c1eac00ac38d741d44

**Comment by 0x4007** [comment]
> @pavlovcik My thoughts on comparing the approaches of Ubiquity and ChainGPT. Ubiquity has a more rational and substantial approach, and ChainGPT is perceived as more innovative and emotional. Certain actions make sense only in the context of a specific approach.

The purpose of this is to translate what they did into the context of what we are doing. 

> 
> A few examples:
> 
> ChainGPT was accelerated by Seedify and this step is crucial for their growth. Ubiquity did not go through acceleration. It would be possible to add the task “Complete acceleration in Seedify” to the actionable plan.

No but instead it would be "Complete acceleration with GitHub" for example. 


> The founder of ChainGPT is public and actively establishes partnerships. It would be possible to add the task “The founder should go public and actively establish partnerships.”

This should be a list of who we should establish partnerships with, based on what ChainGPT did. Besides, during fundraising we will be getting the team out there to meet with people so executing this is feasible. 

> But I don't think it makes sense for Ubiquity to repeat these specific actions, because we have a different approach. The top 5 recommendations that are relevant to us were added in the commit [d7dfe1e](https://github.com/ubiquity/research/commit/d7dfe1ed5a2e28c9e51755c1eac00ac38d741d44)

Those aren't actionable plans. I left comments.

File: chaingpt-research.md [hunk 2]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+### Key actions that can be relevant to us
+
+**1. Establish cooperation with an effective crypto marketing agency**
+
+It is known from open sources that ChainGPT used several marketing agencies.
+In reality, there were probably more agencies than are mentioned in open sources.
+
+- Lead marketing agency - Coinband
+
+https://coinband.io/case/chaingpt
```
Comment (0x4007) [inline chaingpt-research.md:51]: Epic. Thanks for finding this.

File: chaingpt-research.md [hunk 3]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+
+https://www.behance.net/gallery/186506369/ChainGPT-Blockchain-AI-Website 
+
+- One of the additional marketing agencies - EmoneyMax
+
+https://emoneymax.net/  
+
+- ChainGPT itself provides marketing services in the scope of the ChainGPT Pad 
+
+https://medium.com/@chaingpt/chaingpt-pad-2023-a-year-of-unprecedented-success-in-web3-11e7db93615c
```
Comment (0x4007) [inline chaingpt-research.md:67]: Interesting but I'm a bit skeptical. I need to spend more time reading through these carefully. Are there any salient points you want to point out for me?

File: chaingpt-research.md [hunk 4]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+- ChainGPT itself provides marketing services in the scope of the ChainGPT Pad 
+
+https://medium.com/@chaingpt/chaingpt-pad-2023-a-year-of-unprecedented-success-in-web3-11e7db93615c 
+
+Finding other effective agencies that are on the market but have not been used by ChainGPT can be done in separate research.
+
+
+**2. Clearly define the main narrative that we bring to the market**
+
+Alex: will do when producing Series A fundraising materials.
```
Comment (0x4007) [inline chaingpt-research.md:74]: I need to spend more time copying (for a lack of better term) what other crypto AI startups say. You put together some interesting narratives of the other projects in another thread somewhere. When I catch more time to work on the pitchdeck, I'll spend a lot of time studying the other narratives. 

Please feel free to follow along on the pitchdeck progress at https://github.com/ubiquity/series-a/issues/13

Please feel free to leave comments, or fork a copy (don't edit in place!) if you're inspired to edit the deck.

File: chaingpt-research.md [hunk 5]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+**3. Determine the target audience and focus all actions on creating hype and FOMO**
+
+
+**4. Use influencer marketing on a large scale with hundreds of influencers**
+
+Signals of ChainGPT using influencer marketing:
+
+- Massive positive reviews on ProductHunt
+
+https://www.producthunt.com/products/chaingpt/reviews
```
Comment (0x4007) [inline chaingpt-research.md:86]: This is funny to me that they went on producthunt. I assume these are fake reviews, but honestly it doesn't even matter for the purposes of marketing.

File: chaingpt-research.md [hunk 6]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+https://www.linkedin.com/posts/rahulmanglaz_blockchain-web3-ai-activity-7074789416648458240-EKs3
+
+- Massive positive feedback from clients
+
+https://www.quora.com/What-does-ChainGPT-have-to-offer-to-those-in-crypto
+
+There is no information in open sources about the strategy that ChainGPT used in interacting with influencers.
+Influencer marketing is mentioned as one of the activities of the lead marketing agency
+
+>We carefully selected over 47 influencers who were given a four-week timeframe to promote the project and the upcoming IDO. These influencers helped us expand the project's community, resulting in over 133 publications featuring the project.
```
Comment (0x4007) [inline chaingpt-research.md:99]: I can have our lead generation guy do a campaign for this!

File: chaingpt-research.md [hunk 7]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+
+https://coinband.io/case/chaingpt
+
+It doesn't make sense to collect lists of influencers because they usually interact with professional agencies, not companies.
```
Comment (0x4007) [inline chaingpt-research.md:103]: I'm not sure that I agree. If we pay them I'm pretty certain they wouldn't care who's contacting them.

File: chaingpt-research.md [hunk 8]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+
+For small companies like Ubiquity, it is extremely costly and ineffective to collect and maintain a pool of influencers, so they usually collaborate with marketing agencies.
```
Comment (0x4007) [inline chaingpt-research.md:105]: I would like to try.

File: chaingpt-research.md [hunk 9]
```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
...
+
+The advantages of using a marketing agency are already established contacts with many influencers and knowledge of their specifics.
+An effective marketing agency can coordinate influencers and other communication channels into a single campaign that brings results.
```
Comment (0x4007) [inline chaingpt-research.md:108]: This is probably true but I've never had a good experience with a marketing agency yet so if we choose one to work with, ideally it was the same that worked with ChainGPT etc. Basically proven talent.

**Review by 0x4007 — APPROVED** [review]
Great writeup!

**Pull request merged**